feat(sig-services): Add customer zero guideline (#230)
open-services-group/community: sig-services/service-requirements/customer-zero.md

---
title: Have a customer zero
persona: Service developer
phase: Service development
---

**This requirement is optional**

> Having customer zero allows service to get instant feedback before the public release. From customer zero, developers can gather feedback on features of their service, UX design, and others. An important part of this process is not to share any insider information about the project with customer zero. Customer should be allowed to use the service on their own but should be watched by developers and in close contact with support.

Reviewer: ```suggestion
Having customer zero allows service to get instant feedback before the public release. From customer zero, developers can gather feedback on features of their service, UX design, and others. An important part of this process is not to share any developer or maintainer knowledge about the project with customer zero. Customer should be allowed to use the service on their own but should be observed by developers and in close contact with support.
```

## Requirements

> - Customer zero is established; customer zero should not have any insider knowledge of the project and should be allowed to use the service on their own

Reviewer: ```suggestion
- Customer zero should be identified
- Customer zero should not have any developer or maintainer knowledge of the project and should be allowed to use the service on their own
```

Author: Changed in latest commit good point :+1:

## Reasoning

> Using customer zero leads to a better quality of the service before release. Developers can find major flaws in their UX design or problems with the concepts of their service.

Reviewer: ```suggestion
Identifying customer zero leads to a better testing of the service before release. Developers benefit from this testing and can find major flaws in their UX design or problems with the concepts of their service.
```

Author: Fix in latest commit
